backend/picamera_backend.py

```diff
@@ -0,0 +1,92 @@
+from enum import Enum
+from time import time
+from typing import Dict
+
+from backend import Backend, PublishFunc
+from canvas import Canvas
+
+try:
+    from picamera import PiCamera
+    ON_PI = True
```

Comment:
Can you add a comment here to state that this check needs to be located in this file since it's the only/cleanest way of determining if the machine running this is on an RPI? 

Only said this because I was about to ask you to put this check inside `overlay.py` but realised that `overlay.py` does not need the picamera library anyway.

Reply:
Done. Although, I've had an idea, would you rather we check

```
import platform
platform.machine()
```

which will return `armv71` on ARM devices. This is currently how we check if we're on a pi for poetry. Would you rather do this check in `overlay.py` rather than the whole `ON_PI` thing?

Reply:
Oh I didn't know `platform` was part of the python standard library. I would actually prefer this. To me it feels like it's a more elegant solution rather than catching an exception if we can't import a single library. Up to you if you want it to be in the same or different PR. I will accept this PR anyway.

Reply:
Done!